frontend/src/components/header.tsx

```diff
@@ -2,6 +2,8 @@ import React from 'react';
 
 import styled from '@emotion/styled';
 import MenuIcon from '@mui/icons-material/Menu';
+import SearchIcon from '@mui/icons-material/Search';
+import { InputBase } from '@mui/material';

```

Comment:
```suggestion
import InputBase from '@mui/material/InputBase';
```
위의 형태로 바꿔주시는게 convention이 맞아보입니다.
아직 우리가 정하지 않은 rule인데 어떻게 할지 다른 open source를 찾아보고
다 같이 정한 후 전체적인 수정여부를 결정합시다.